### PR TITLE
Fix spelling of WASM and Web Assembly

### PR DESCRIPTION
**Pull request by webmaster128**
# Description

"The pain is real."

# Review

- [ ] Add a short description of the the change to the CHANGELOG.md file

### CODE REVIEW

**Review by jubianchi — APPROVED** [review]

File: tests/wast/spec/proposals/threads/memory.wast [hunk 1]
```diff
@@ -95,7 +95,7 @@
 
 (module
   (memory 1)
-  (data (i32.const 0) "ABC\a7D") (data (i32.const 20) "WASM")
+  (data (i32.const 0) "ABC\a7D") (data (i32.const 20) "Wasm")
```
Comment (syrusakbary) [inline tests/wast/spec/proposals/threads/memory.wast:98]: This made the tests to fail.

```suggestion
  (data (i32.const 0) "ABC\a7D") (data (i32.const 20) "WASM")
```

File: tests/wast/spec/memory.wast [hunk 2]
```diff
@@ -91,7 +91,7 @@
 
 (module
   (memory 1)
-  (data (i32.const 0) "ABC\a7D") (data (i32.const 20) "WASM")
+  (data (i32.const 0) "ABC\a7D") (data (i32.const 20) "Wasm")
```
Comment (syrusakbary) [inline tests/wast/spec/memory.wast:94]: This made the tests to fail.

```suggestion
  (data (i32.const 0) "ABC\a7D") (data (i32.const 20) "WASM")
```

**Pull request merged**